search-app/src/cmr/search/results_handlers/results_handler_util.clj

```diff
@@ -0,0 +1,61 @@
+(ns cmr.search.results-handlers.results-handler-util
+  "Provides useful functions for several of the result handlers."
+  (:require
+   [clojure.set :as s]
```

Comment:
there is an effort to get rid of the single letter import names

Reply:
set is hard, because I can't use set.  I'll stick with s for now and see if I can use a different term in the future.